src/App.tsx

```diff
@@ -1,14 +1,49 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { SortBy } from './types/SortBy';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | []>([]);
+  const [query, setQuery] = useState('');
+  const [sortBy, setSortBy] = useState('all');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  const loadTodos = async () => {
+    setTodos(await getTodos());
+  };
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  const filtredTodos = todos.filter(todo => {
+    const formatedQuery = query.trim().toLowerCase();
+    const titleIncludesQuery = todo.title.toLowerCase().includes(formatedQuery);
```

Comment:
todos filtered even query is empty, maybe you can optimize it, add some condition)